Header header
float32 chip_temp
float32[64] pixel_temps
 
